moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -767,7 +767,8 @@ bool JogCalcs::updateJoints(JogArmShared& shared_variables)
         }
         else
         {
-          ROS_WARN_STREAM_NAMED(LOGNAME, "An acceleration limit is not defined for this joint; minimum stop distance "
+          ROS_WARN_STREAM_THROTTLE_NAMED(1.0, LOGNAME,
+                                         "An acceleration limit is not defined for this joint; minimum stop distance "
                                          "should not be used for collision checking");
```

Comment:
Personally, I think this should be changed to ROS_ERROR_STREAM_ONCE_NAMED(). I think one fat red message is enough.

Reply:
Maybe also add the joint name so we can see what joint is the issue if it is only one:

```
ROS_ERROR_STREAM_ONCE_NAMED(LOGNAME,
                            "An acceleration limit is not defined for joint: "
                                << joint_model->getName()
                                << "; minimum stop distance should not be used for collision checking.")
```

Reply:
Done.